### DISCUSSION TITLE
Why IfcBridgePart is not implemented in "FC4X3_RC4" Ifcopenshell version

### DISCUSSION THREAD

**Discussion by jeangual** [IfcOpenShell/IfcOpenShell#4341] [Q&A]
In the last version of IFC ( IFC 4.3.2.0 (IFC4X3_ADD2) ) IfcBridgePart inherit from IfcFaciltyPart but in IfcOpenshell it is not possible to define IfcBridgePart for this IFC version. Is it a know issue? When this entity will be define?

**Comment by aothms** [comment]
See https://github.com/buildingSMART/IFC4.3.x-development/issues/90

The various facility parts were previously handled using a union of enumerations.

The latest version to use is IFC4X3_ADD2. Not Release Candidate 4.

**Comment by RickBrice** [comment]
I think IfcBridgePart is valid for IFC4.3Add2  https://standards.buildingsmart.org/IFC/RELEASE/IFC4_3/HTML/lexical/IfcBridgePart.htm

**Reply by jeangual** [reply]
@Aothms: thank you very much but when i try:
  ifc_file = ifcopenshell.file(schema='IFC4X3_ADD2') in the python code I have an error warning (with the latest version of openshell).
@RickBrice: Yes It is true but my problem is Ifcopenshell does not accept IfcBridgePart entity.

**Reply by KevinHuot** [reply]
@jeangual : Update your version of IfcOpenShell, the last one from PyPI fixes the issue (v0.7.0-6c9e130ca).

**Reply by jeangual** [reply]
Thanks everybody. I  will try this new version of ifcopenshell.

**Comment by RickBrice** [comment]
@jeangual 
I'm not much of a python developer but I see this which seems to indicate IfcBridgePart is in IfcOpenShell

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/37087370/e1b05cff-6b4d-4e2e-b946-36062e89e739)

**Reply by KevinHuot** [reply]
In fact the pre-built packages version raises this error :
![ErrorPreBuilt](https://github.com/IfcOpenShell/IfcOpenShell/assets/160717828/a2933112-f81a-4138-ba5b-7609d8b34abc)
Which is possible according to the warnings in the description : https://blenderbim.org/docs-python/ifcopenshell-python/installation.html

**Comment by jeangual** [comment]
Thanks every body. All is fine with the latest release of ifcopenshell (python version) : (v0.7.0-6c9e130ca)
